.github/workflows/jest.yml

```diff
@@ -2,12 +2,13 @@ name: jest
 
 on:
   push:
+    branches: [main]
   pull_request:
+    branches: [main]
 
 jobs:
   jest:
     runs-on: ubuntu-latest
-    if: ${{ github.event_name != 'pull_request' || github.repository_owner != github.event.pull_request.head.repo.owner.login }}
```

Comment:
Can we keep the logic for the Jest workflow?

Reply:
yes, do not revert this